CODEOWNERS

```diff
@@ -41,6 +41,7 @@ Makefile*                                                        @istio/wg-test-
 /pkg/tracing/                                                    @istio/wg-policies-and-telemetry-maintainers
 /pkg/wasm/                                                       @istio/wg-policies-and-telemetry-maintainers
 /pkg/webhooks/                                                   @istio/wg-environments-maintainers
+/pkg/workloadapi/                                                @istio/wg-networking-maintainers
```

Comment:
Should workloadapi/security be owned by @istio/wg-security-maintainers?

Reply:
seems reasonable to me

Reply:
Maybe co-ownership? Specifically because the changes to that directory need to be replicated in ztunnel